cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -299,7 +299,7 @@
           },
           "InstancesDistribution": {
             "OnDemandBaseCapacity": 0,
-            "OnDemandPercentageAboveBaseCapacity": 100,
+            "OnDemandPercentageAboveBaseCapacity": ${100 - (group.spotPercentage!0)},
```

Comment:
I think we should use here the OnDemandPercentage and calculate the value in the java class

Reply:
well, yes, let's do it that way